basic/payment-splitter/src/lib.rs

```diff
@@ -0,0 +1,204 @@
+use scrypto::prelude::*;
+
+/// Defines the data of the shareholders badge which is an NFT.
+#[derive(NftData)]
```

Comment:
Objects related to NFTs were renamed in v0.3. Please look at https://docs.radixdlt.com/main/scrypto/release_notes/v0_3.html#_changed_naming_of_nft_related_objects_and_methods